android-client-sdk/src/main/java/com/devcycle/sdk/android/api/DVCClient.kt

```diff
@@ -61,9 +64,23 @@ class DVCClient private constructor(
             try {
                 fetchConfig(user)
                 isInitialized.set(true)
-                withContext(Dispatchers.IO){
-                    eventSource = EventSource.Builder(Handler(fun () {}), URI(config?.sse?.url)).build()
-                    eventSource.start()
+                if (config?.sse?.url != null) {
+                    withContext(Dispatchers.IO) {
+                        eventSource = EventSource.Builder(Handler(fun(messageEvent: MessageEvent?) {
+                            val data = JSONObject(messageEvent?.data)
```

Comment:
should this just early return if the messageEvent is null? Also is that even possible?

Reply:
i can just wrap all the other logic in an if statement checking that it's not null, so it does nothing if the message event is null for some reason